client/src/featureform/resources.py

```diff
@@ -578,20 +578,24 @@ def _create(self, stub) -> None:
         stub.CreateTrainingSetVariant(serialized)
 
     def _create_local(self, db) -> None:
+        self._insert_training_set_features(db)
+        try:
+            db.getNameVariant("labels_variant", "labelName", self.label[0], "variantName", self.label[1])
+        except ValueError:
+            raise ValueError(self.label[0] + " does not exist. Failed to register training set")
```

Comment:
Typically cleaner to do something like:
"Failed to register training set. Label does not exist: {}".format(label[0])